<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotEditorWidget</class>
 <widget class="QWidget" name="BotEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="PropertiesView" name="bot_params_view"/>
      <widget class="QWidget" name="layoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Bot stdout output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="stdout_textedit">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Bot stderr output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="stderr_textedit">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="BotView" name="graphics_view">
      <property name="styleSheet">
       <string notr="true">QGraphicsView
{
border:none;
background:transparent;
}</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BotView</class>
   <extends>QGraphicsView</extends>
   <header>constructor_app.graphic_scene.bot_view</header>
  </customwidget>
  <customwidget>
   <class>PropertiesView</class>
   <extends>QTreeView</extends>
   <header>common.properties_view</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
